cms/migrations/0031_merge_20230817_1538.py

```diff
@@ -0,0 +1,12 @@
+# Generated by Django 3.2.20 on 2023-08-17 15:38
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("cms", "0030_add_faq_remove_faculty_reorder_things"),
```

Comment:
Could you recreate this migration with number 0031 so that the merge migration is not needed?

Reply:
Done in https://github.com/mitodl/mitxonline/pull/1816/commits/6775405dcd4bdf9dc8dd8962dd718b11540718d7